google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreOptions.java

```diff
@@ -145,13 +217,44 @@ public FirestoreRpcFactory getDefaultRpcFactory() {
     @Nonnull
     @Override
     public TransportOptions getDefaultTransportOptions() {
-      return getDefaultGrpcTransportOptions();
+      return TRANSPORT_OPTIONS;
+    }
+
+    @Nonnull
+    TransportChannelProvider getTransportChannelProvider() {
```

Comment:
`getDefaultTransportChannelProvider`? (or, the previous one should be `getTransportOptions` - which could make sense since this is already in a `FirestoreDefaults` class)

Reply:
The other methods are inherited, so I changed the name to use "Default" everywhere.